package.json

```diff
@@ -80,10 +82,11 @@
     "react-monaco-editor": "^0.7.3",
     "react-newline-to-break": "^1.0.6",
     "react-rectangle": "1.3.3",
-    "react-resumable-js": "https://github.com/aaronpcz/ReactResumableJS#1.1.27",
+    "react-resumable-js": "git+https://github.com/aaronpcz/ReactResumableJS.git#1.1.27",
```

Comment:
Only way I could get this to work, break anything? @aaronpcz

Reply:
Should be fine - interesting - must be some difference between npm version used.

Not super important but post your node + npm version.
I think we should specify in the package json the engines, allowing us to share the same node and npm version.

Reply:
I bet the later versions of npm require the git+

The earlier versions of npm had a lot to learn from other dependency management tools.  Looks like they are slowly but surely getting there.